docker/env/rep.env

```diff
@@ -17,8 +17,8 @@ OAUTH_AUTHORITY_HOST_URL=https://login.microsoftonline.com/
 OAUTH_TENANT=6f504113-6b64-43f2-ade9-242e05780007
 
 # Power Apps
-MSPA_SCOPE=https://sdds-dev.crm11.dynamics.com/.default
-MSPA_CLIENT_URL=https://sdds-dev.crm11.dynamics.com/api/data/v9.0
+MSPA_SCOPE=https://defra-sdds-dev2.crm11.dynamics.com/.default
+MSPA_CLIENT_URL=https://defra-sdds-dev2.crm11.dynamics.com/api/data/v9.0
```

Comment:
Not sure why these are changing? `sdds-dev` should still be the default dev environment.

But also, I'm slightly dubious about these being in the repo in the first place. I know they're not really "secrets" like credentials but they still form part of the information being shared through the authentication/authorisation journey so I'd prefer that they were held elsewhere unless there's a really good reason not to.

Reply:
Apparently another environment has been created, and this is now the development environment. I thought that including this would make setup easier for Del, but I get that they should not really be in the repo - so removed as requested.